Makefile

```diff
@@ -0,0 +1,52 @@
+# Some simple testing tasks (sorry, UNIX only).
+
+FLAGS=
+KAFKA_VERSION=0.11.0.0
+SCALA_VERSION=2.11
+
+setup:
+	pip install -r requirements-dev.txt
```

Comment:
probably want a virtualenv here, dont we? or perhaps --user ?

Reply:
I would prefer to assume the virtualenv is already installed and activated. There are a lot of different ways to do that, like pyvenv, virtualenv, virtualenvwrapper and so on.

Reply:
I can change the name to something like `setup-local`